gyro_light.py

```diff
@@ -12,32 +12,25 @@
 # Initializes L3GD20 object using default range, 250dps
 SENSOR = my_l3gd20.L3GD20_I2C(I2C, address=0x6B)
 
+# SPEED = 1  # Increase to slow down the rainbow. Decrease to speed it up.
+
 num_pixels = 95  # Update this to match the number of LEDs.
-SPEED = 1  # Increase to slow down the rainbow. Decrease to speed it up.
+sleep_interval_seconds = 0.05 # time in seconds to sleep between each pixel light up.
 pixel_pin = board.GP26  # This is the default pin on the 5x5 NeoPixel Grid BFF.
 ORDER = neopixel.GRB
 
-pixels = neopixel.NeoPixel(board.GP26, 95)
-
+pixels = neopixel.NeoPixel(board.GP26, num_pixels)
 
-i = 0
-pix_position = i % len(pixels)
-reset_led = False
+pixel_index = 0
 

```

Comment:
`pixel_pos = num_pixels - 1 # [pixel position] there are 95 pixels but n-1 needed as pixel number 0 through 94 is counted`